src/components/Display.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+class Display extends React.Component {
+  constructor(props) {
+    super(props);
+    const { resultDefault } = this.props;
+    this.state = { result: resultDefault };
+  }
+
+
+  render() {
+    return (
+      <div>
+        <input type="text" value={this.state.result} />
+      </div>
+    );
+  }
+}
+
+
+export default Display;
```

Comment:
- use `defaultProps` to make `resultDefault` to be of value "0"  [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]
- use `propTypes` to make `resultDefault` to be of type "STRING"  [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]